hw/ip_templates/otp_ctrl/dv/env/otp_ctrl_scoreboard.sv.tpl

```diff
@@ -1406,6 +1604,19 @@ else:
     return digest;
   endfunction
 
+  // Expect $countones of each 32-bit words to be at least 28.
+  virtual function bit is_zeroized(int part_idx);
+    if (part_is_zeroizable(part_idx)) begin
+      bit [TL_DW-1:0] zeroized_addr = PART_OTP_ZEROIZED_ADDRS[part_idx];
+      // TODO: Change the magic constant 28 to a Parameter defined in the pkg that is used by the
```

Comment:
This is in https://github.com/lowRISC/opentitan/blob/master/hw/ip_templates/otp_ctrl/rtl/otp_ctrl_top_specific_pkg.sv.tpl#L87, and applies to the 64 bits, so you could add countones of addr and addr+1, and compare that to ZeroizationValidBound.